src/iOS.Core/Controllers/BaseLockPasswordViewController.cs

```diff
@@ -54,12 +54,15 @@ public BaseLockPasswordViewController(IntPtr handle)
         public abstract UIBarButtonItem BaseSubmitButton { get; }
         public abstract Action Success { get; }
         public abstract Action Cancel { get; }
+        public Action LaunchHomePage;
```

Comment:
⛏️ Transform into property instead of field
```suggestion
        public Action LaunchHomePage { get; set; }
```